src/MoBi.Presentation/Presenter/Main/SimulationExplorerPresenter.cs

```diff
@@ -161,6 +161,11 @@ protected override IContextMenu ContextMenuFor(ITreeNode treeNode)
          if (treeNode.TagAsObject is ClassifiableSimulation simulation)
             return ContextMenuFor(new SimulationViewItem(simulation.Simulation));
 
+         if (treeNode.TagAsObject is IBuildingBlock buildingBlock)
+         {
```

Comment:
to keep consistent with the code avobe, remove the { } 
in general,  I'd try to use newer terser syntax if possible. ForInstance on line 74 above, remove the redundant { } and return. as well

public bool IsSatisfiedBy(IViewItem viewItem, IPresenterWithContextMenu<IViewItem> presenter)
      {
         return viewItem is SimulationBuildingBlockViewItem;
      }

public bool IsSatisfiedBy(IViewItem viewItem, IPresenterWithContextMenu<IViewItem> presenter)
      => viewItem is SimulationBuildingBlockViewItem;